src/components/ComponentDocsList/ComponentDocsList.sass

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/core'
+
+.ComponentDocsList
+    margin-left: 32rem
+    padding: 15px
+    background-color: white
+    min-height: calc(100vh - $nav-height)
```

Comment:
Is this actually compiling out the way you expect? I always have issues when using a sass variable inside of `calc()` - it usually outputs the literal text (`calc(100vh - $nav-height)`), which CSS ignores.